test/Microsoft.AspNetCore.SignalR.Client.FunctionalTests/Startup.cs

```diff
@@ -25,6 +25,7 @@ public void ConfigureServices(IServiceCollection services)
                 options.EnableDetailedErrors = true;
             })
                 .AddMessagePackProtocol();
+            services.AddSingleton<IUserIdProvider, HeaderUserIdProvider>();
```

Comment:
If we ever want to test something else, we're fucked 😄

Reply:
Not really...